gateway/src/bin/mgs.rs

```diff
@@ -70,26 +71,24 @@ async fn do_run() -> Result<(), CmdError> {
     let args = Args::parse();
 
     match args {
-        Args::Openapi => run_openapi().map_err(CmdError::Failure),
+        Args::Openapi => {
+            run_openapi().map_err(|e| CmdError::Failure(anyhow!("{e}")))
```

Comment:
Should this be

```suggestion
            run_openapi().map_err(|e| CmdError::Failure(anyhow!(e)))
```

instead? That's the form used down on line 106, but I'm not sure what the difference is off the top of my head.

I'll leave the same suggestion other places I see it under the assumption that it's nicer to have `anyhow` wrapping an actual error than wrapping an error converted to a string; I _think_ as written we might lose any further error sources in the chain, because anyhow would just see a string here?

I also see a fair amount of `CmdError::Failure(e.into())`, which I think is equivalent to `CmdError::Failure(anyhow!(e))`. I don't mind both of those being present; I have a very, very slight preference for `anyhow!(e)` over `e.into()` but wouldn't bother changing the `e.into()`s.